test/bin/composer_cleanup.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+#
+# This script cancels any composer jobs and deletes failed and completed jobs.
```

Comment:
Should we also spin down the worker services here?

Reply:
They don't take up many resources when there are no images to build.

Reply:
Still, it might be a logical thing to do here, right?